doc/adding_users.rst

```diff
@@ -1,7 +1,14 @@
 Requesting Lab Access
 =====================
 
-Send a lab admin an email with two pieces of information:
+Send a lab admin an email with these pieces of information:
+
+#. Whether you are only requesting access to schedule jobs (and view their
+   results), or also to run one-off jobs. The latter is more restricted in that
+   it is usually only granted to employees.
```

Comment:
and maybe say 'run teuthology jobs manually' instead of one-off.. it took mea  minute to figure out what you meant